src/components/00-materials/styles/breakpoint/_helpers.scss

```diff
@@ -5,18 +5,30 @@
   $value-unit: unit($value);
 
   // Will convert relative EMs into root EMs.
-  @if breakpoint-get('base font size') and type-of(breakpoint-get('base font size')) == 'number' and $value-unit == 'em' {
+  @if breakpoint-get('base font size') and
+    type-of(breakpoint-get('base font size')) ==
+    'number' and
+    $value-unit ==
+    'em'
+  {
     $base-unit: unit(breakpoint-get('base font size'));
 
-    @if $base-unit == 'px' or $base-unit == '%' or $base-unit == 'em' or $base-unit == 'pt' {
-      @return base-conversion($value) / base-conversion(breakpoint-get('base font size')) * 1em;
-    }
-    @else {
-      @warn '#{breakpoint-get(\'base font size\')} is not set in valid units for font size!';
+    @if $base-unit ==
+      'px' or
+      $base-unit ==
+      '%' or
+      $base-unit ==
+      'em' or
+      $base-unit ==
+      'pt'
+    {
```

Comment:
Crazy

Reply:
Yeah this looks pretty stupid to me too. This is what prettier does to javascript too if the lines get too long.

Reply:
Then we should either change the line length of prettier (for .scss files) or use a prettier-ignore in those cases.

We are not slaves to the machine, so let's take control here (and elsewhere).